1/src/Triangle.ts

```diff
@@ -0,0 +1,34 @@
+import {Point} from "./point";
+import {Shape} from "./shape";
+
+export class Triangle extends Shape {
+    constructor(point1: Point, point2: Point, point3: Point);
+    constructor(point1: Point, point2: Point, point3: Point, color: string, filled: boolean);
+    constructor(point1: Point, point2: Point, point3: Point, color?: string, filled?: boolean) {
+        super([point1, point2, point3], color, filled);
+    }
+
+    public toString(): string {
+        let resultString: string = `Triangle[`;
+
+        this.points.forEach((point: Point, index: number) => {
+            resultString += `v${index + 1}=${point.toString()}${index === this.points.length - 1 ? ']' : ','}`
+        });
+
+        return resultString;
```

Comment:
Instead of forEach loop you can just have one-line function. 
```
return `Triangle[v1=${this.point1.toString()}, v2=${this.point2.toString()}, v3=${this.point3.toString()}]`
```